slp_visio/tests/unit/load/strategies/dataflow/impl/test_create_connector_by_line_coordinates.py

```diff
@@ -0,0 +1,96 @@
+from unittest.mock import MagicMock, Mock
+
+from pytest import mark
+
+from slp_visio.slp_visio.load.strategies.connector.impl.create_connector_by_line_coordinates import \
+    CreateConnectorByLineCoordinates
+
+
+def mock_component(_id, pos):
+    x, y = pos[0], pos[1]
+    width, height = pos[2], pos[3]
+    return MagicMock(ID=_id, begin_x=x, begin_y=y,
+                     cells={'Width': Mock(value=width), 'Height': Mock(value=height)},
+                     center_x_y=(float(x) + float(width) / 2, float(y) + float(height) / 2))
+
+
+class TestCreateConnectorByLineCoordinates:
+
+    @mark.parametrize('line,start,end', [
+        (['1040', '-560', '1290', '-560'], ['960', '-600', '80', '80'], ['1290', '-590', '60', '60']),
```

Comment:
Could we add an id to each `param` to make easier to understand what is going on in each test? For example:
* id="start connected, end not connected"
* id="end not connected, end connected"
* id="both disconnected"
* ...
At a first glance, it looks like a very nice and complete test, but, if something fails, it is difficult to identify what is exactly the use case.